### ISSUE TITLE
Honkalculate doesn't work

### ISSUE THREAD

**Issue by turskain** [Zarel/honko-damagecalc#159]
Pressing Honkalculate with any tier checked prints the following error in console:

Uncaught TypeError: pokeInfo.find is not a function
    at new Pokemon (shared_controls.js:408)
    at calculate (honkalculate_controls.js:76)
    at HTMLButtonElement.<anonymous> (honkalculate_controls.js:231)
    at HTMLButtonElement.dispatch (jquery-1.9.1.min.js:3)
    at HTMLButtonElement.v.handle (jquery-1.9.1.min.js:3)

**Comment by MagiaSN** [comment]
The code in this repository works well in my browser, but the page https://pokemonshowdown.com/damagecalc/honkalculate.html?mode=one-vs-all is broken. It seems the code of

**Comment by Zarel** [comment]
This is probably why I wanted a cachebuster...

**Comment by AustinXII** [comment]
@Zarel I'm pretty busy with work right now, feel free to add it yourself :p

**Comment by Zarel** [comment]
hashtag not my damagecalc

**Comment by AustinXII** [comment]
No but tbf it is PS's which is yours

**Comment by Zarel** [comment]
The agreement was that I'd host it for Honko and he'd maintain it, and then he dropped off the face of the earth.

I really don't want to maintain it. :(

I guess I'll do the cachebuster, though.

**Comment by AustinXII** [comment]
Thanks Zarel, you don't have to worry about maintaining it but helping out with adding features you want is always appreciated

**Comment by Honko** [comment]
I've returned to the face of the earth; I just have no interest in continuing to maintain my crappy several-year-old code. I do enough of that at work...

For the record, I don't think I'll ever pick this back up (other than sending occasional setdex updates). I started this because I wanted to build a fast, convenient, all-gen calc and learn a little js for fun. I did that, and now I've moved on.

@Zarel Do you have any long-term plans for replacing this calc? I think I vaguely remember you saying a long time ago that you eventually wanted to have damage calc functionality available directly into the battle UI, which would eliminate any need for a separate PS-hosted calc. Idk if that's still an idea you're considering. If not, I wonder if it's worth adding an issue in the PS queue for someone to build a new standalone calc? I know you get new devs wanting to contribute, and this seems like a nice project for someone who knows what they're doing but doesn't require a lot of familiarity with the code of PS itself. Austin's done an admirable job keeping this calc alive and up-to-date, but it would probably be better for everyone involved to have a calc that's written to PS's standards by someone who knows modern js better than I did in 2012. Just my 2c.

tl;dr thanks for cleaning up my mess

**Comment by AustinXII** [comment]
I still think that is planned and yeah I agree that would be much better.

**Comment by Zarel** [comment]
I still have long-term plans for adding better tooltip stuff.

I think there's still a use for a standalone calculator (if only for in-game stuff). I can add it to Ideas for New Developers.

**Comment by AustinXII** [comment]
#166
